tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -182,4 +187,29 @@ describe(__filename, () => {
     expect(root.find('.AddonMoreInfo-database-id-content'))
       .toHaveText('9001');
   });
+
+  it('links to stats if add-on author is viewing the page', () => {
+    const addon = {
```

Comment:
This should be `addon = createInternalAddon({ ...fakeAddon, slug: ... })`.

Also, while you're in there could you fix the [`render()` ](https://github.com/mozilla/addons-frontend/blob/e3f014c003ca6153da56688abe2658304f3ac30f/tests/unit/amo/components/TestAddonMoreInfo.js#L24) function?

```js
function render(props) {
    return shallowUntilTarget(
      <AddonMoreInfo
        addon={createInternalAddon(fakeAddon)}
        i18n={getFakeI18nInst()}
        store={store}
        {...props}
      />,
      AddonMoreInfoBase
    );
}
```